### ISSUE TITLE
ansible-lint does not respect yamllint error level

### ISSUE THREAD

**Issue by Hipska** [ansible/ansible-lint#4355]
##### Summary

Rules for yamllint set to warning aren't automatically warnings in ansible-lint as well. It only does that when the rule is also added to `warn_list`, which is duplicating configs.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console
ansible-lint --version
```

```
ansible-lint 24.9.2 using ansible-core:2.17.4 ansible-compat:24.9.1 ruamel-yaml:0.18.6 ruamel-yaml-clib:0.2.8
```

- ansible installation method: OS package
- ansible-lint installation method: OS package

##### STEPS TO REPRODUCE

.yamllint

```yaml
---

rules:
  line-length:
    level: warning
    allow-non-breakable-inline-mappings: true
```

##### Desired Behavior

```
yaml[line-length]: Line too long (413 > 80 characters) (warning)
initiate.yml:25

yaml[line-length]: Line too long (413 > 80 characters) (warning)
initiate.yml:29
```

##### Actual Behavior

```
yaml[line-length]: Line too long (413 > 80 characters)
initiate.yml:25

yaml[line-length]: Line too long (413 > 80 characters)
initiate.yml:29
```

**Comment by Qalthos** [comment]
This seems like an ask for reasonable behavior. We do look at .yamllint, so we could use that to inform the level of yaml rules, though I'm not sure how complicated that would be.

**Comment by Hipska** [comment]
Even disable yaml rules does not seem to be respected..

```yaml
---

rules:
  truthy: disable
```
Results:

```
yaml[truthy]: Truthy value should be one of [false, true] (warning)
handlers/main.yml:5
```
